ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py

```diff
@@ -182,3 +182,50 @@ def test_unravel_index(
         indices=np.asarray(x[0], dtype=input_dtype[0]),
         shape=shape,
     )
+
+    
+# nanquantile
+
+@handle_test(
+    fn_tree="functional.experimental.nanquantile",
+    dtype_x_axis=statistical_dtype_values(function="nanquantile"),
```

Comment:
you are using `nanquantile`, there is no condition in the function which handles `nanquantile()` in `statistical_dtype_values()`, there is handling for `quntile()`, you can use that, as the both the functions accepts the same inputs. You were also generating array of integers which is not accepted by torch and it was causing an error, this will also fix that.
This should be statistical_dtype_values(function="quantile")